doc/source/data/doc_code/hf_quick_start.py

```diff
@@ -0,0 +1,51 @@
+# flake8: noqa
+# isort: skip_file
+# fmt: off
+
+# __hf_quickstart_load_start__
+import ray
+import pandas as pd
+
+
+prompts = pd.DataFrame(["Complete these sentences", "for me"], columns=["text"])
+ds = ray.data.from_pandas(prompts)
+# __hf_quickstart_load_end__
+
+
+# __hf_quickstart_model_start__
+class HuggingFacePredictor:
+    def __init__(self):  # <1>
+        from transformers import pipeline
+        self.model = pipeline("text-generation", model="gpt2")
+
+    def __call__(self, batch):  # <2>
+        return self.model(list(batch["text"]), max_length=20)
+# __hf_quickstart_model_end__
+
+
+# __hf_quickstart_prediction_start__
+scale = ray.data.ActorPoolStrategy(2)
```

Comment:
```suggestion
scale = ray.data.ActorPoolStrategy(size=2)
```

Kwargs will be required in strict mode for actor pool strategy.